qiskit/visualization/latex.py

```diff
@@ -235,44 +220,35 @@ def _initialize_latex_array(self):
                 self._latex[i][0] = label
 
     def _get_image_depth(self):
-        """Get depth information for the circuit.
-
-        Returns:
-            int: number of columns in the circuit
-            int: total size of columns in the circuit
-        """
-
+        """Get depth information for the circuit."""
         max_column_widths = []
-        # Determine row spacing before image depth
+        # Determine wire spacing before image depth
         for layer in self.ops:
             for op in layer:
-                # useful information for determining row spacing
-                boxed_gates = ['u0', 'u1', 'u2', 'u3', 'x', 'y', 'z', 'h', 's',
-                               'sdg', 't', 'tdg', 'rx', 'ry', 'rz', 'ch', 'cy',
-                               'crz', 'cu3', 'id']
+                # useful information for determining wire spacing
+                boxed_gates = ['u1', 'u2', 'u3', 'u', 'p', 'x', 'y', 'z', 'h',
+                               's', 'sdg', 't', 'tdg', 'sx', 'sxdg', 'rx', 'ry', 'rz',
+                               'ch', 'cy', 'crz', 'cu2', 'cu3', 'cu', 'id']
                 target_gates = ['cx', 'ccx', 'cu1', 'cp', 'rzz']
                 if op.name in boxed_gates:
                     self.has_box = True
-                if op.name in target_gates:
-                    self.has_target = True
-                if isinstance(op.op, ControlledGate):
+                elif op.name in target_gates:
                     self.has_target = True
+                elif isinstance(op.op, ControlledGate):
+                    self.has_box = True
 
         for layer in self.ops:
 
-            # store the max width for the layer
```

Comment:
why not keeping these comments?

Reply:
See comment below about the _get_image_depth function.